tests/unit/amo/components/TestAddon.js

```diff
@@ -804,58 +791,6 @@ describe(__filename, () => {
       .toHaveProp('defaultInstallSource', INSTALL_SOURCE_DETAIL_PAGE);
   });
 
-  it('enables a theme preview for non-enabled add-ons', () => {
```

Comment:
There still seems to be a test in this file for `it('toggles a theme on click')`, which I think can be removed. I guess it will start to fail when the other properties are removed from the component.

Reply:
yep - there is definitely more pieces to this than i originally thought. will take another look to remove ALL related preview parts here